rasa_nlu/classifiers/sklearn_intent_classifier.py

```diff
@@ -178,7 +178,7 @@ def process(self, message, **kwargs):
         else:
             X = message.get("text_features").reshape(1, -1)
             intent_ids, probabilities = self.predict(X)
-            intents = self.transform_labels_num2str(intent_ids)
+            intents = self.transform_labels_num2str(intent_ids.T)
```

Comment:
is that correct or should it be `np.ravel(intent_ids)`?

Also, after this change, what happens if we run with 0.19, does it still work?

Reply:
I've included the change, but we flatten this vector in the following line so it's not an issue
Just ran the script with 0.19, it still passes

Reply:
awesome, thanks :+1: